### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by nsbendre** [ildoonet/tf-openpose#6]
Hi, 
I am trying to train your model from scratch. I dont exactly get it how you have fed the data to the model. Can you please provide the steps ? 
Thanks!

**Comment by ildoonet** [comment]
I wrote a simple guide on README : https://github.com/ildoonet/tf-openpose#training

Let me know after you try.

**Comment by nsbendre** [comment]
Hi,
I think my question was confusing. let me rephrase, '/data/public/rw/coco-pose-estimation-lmdb/' How did you download/generate data at this path ?? 

Will surely keep you in loop once I begin with my training.

Thanks!

**Comment by felixfuu** [comment]
@ildoonet Can you share a model that you have trained on coco? Then i can evaluate it and train it on coco on the basis of  you model.

**Comment by ildoonet** [comment]
@nsbendre 

I updated README. 

```
$ wget -nc --directory-prefix=lmdb_trainVal/ 		http://posefs1.perception.cs.cmu.edu/Users/ZheCao/lmdb_trainVal/data.mdb
$ wget -nc --directory-prefix=lmdb_trainVal/ 		http://posefs1.perception.cs.cmu.edu/Users/ZheCao/lmdb_trainVal/lock.mdb
```

@felixfuu I will but no at the moment. It will take few weeks before I release them.

**Comment by felixfuu** [comment]
@ildoonet  Thank you ~

**Comment by andyqian2015** [comment]
Hi，@felixfuu  ,Have you finished trained the model？ does the loss converage ?

**Comment by andyqian2015** [comment]
thx if you can reply me
